source/operations/concepts/erasure-coding.rst

```diff
@@ -10,62 +10,131 @@ Erasure Coding
+Erasure Coding Basics
+---------------------
+
+   .. note::
+      
+      The diagrams in this section present a simplified view of MinIO erasure coding operations and are not intended to represent the complexities of MinIO's full erasure coding implementation.
+
+MinIO implements Erasure Coding as a core component in providing data redundancy and availability.
+   Erasure Coding involves partitioning an object into **data** and **parity** shards.
+
+   .. figure:: /images/erasure/erasure-coding-shard.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of an object being sharded using MinIO's Reed-Solomon Erasure Coding algorithm.
+
+      A client writes an object to MinIO.
+      MinIO automatically partitions the object into data and parity shards.
+
+An object's :term:`Parity` dictates how many parity shards MinIO creates for an object.
+   MinIO uses the :term:`erasure set size <erasure set>` and configured parity for the deployment to determine the number of parity and data shards to create for each object, where:
+
+   :math:`N (ERASURE\ SET\ SIZE) = K (DATA) + M (PARITY)`
+
+   .. figure:: /images/erasure/erasure-coding-shard-ec6.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of an object written with EC:6 to an erasure set with 16 drives
+
+   This deployment has an erasure set size of ``16`` and a parity of ``EC:6``. 
+   Based on the formula, MinIO creates ``10`` data shards and ``6`` parity shards for each object.
+
+MinIO can use any parity shard to reconstruct any data shard.
+   .. figure:: /images/erasure/erasure-coding-shard-reconstruction.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of MinIO using parity shards to reconstruct data shards
+
+      An object has lost two data shards.
+      MinIO uses any two of the available parity shards to reconstruct the data shards.
+      Even a single remaining parity shard would be sufficient to reconstruct both missing data shards.
+
+MinIO requires a minimum of ``K`` Data *or* Parity shards to **read** the object.
+   The value ``K`` constitutes the **read quorum** for the deployment.
+   Alternatively, the object can tolerate the loss of no more than ``M`` shards of any time while remaining readable.
+
+   .. figure:: /images/erasure/erasure-coding-shard-read-quorum.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of an object maintaining read quorum with only 10 total shards remaining
+
+      An object has 10 data shards and 6 parity shards for a total of 16 shards.
+      It has lost 3 data and 3 parity shards for a total of 6 shards lost and 10 remaining.
+      The loss of any further shards would result in losing read quorum.
+
+   An object that has lost read quorum is effectively "lost" and may be recovered through other means such as :ref:`replication resynchronization <minio-bucket-replication-resynchronize>`.
```

Comment:
perhaps

`"lost" and cannot be directly recovered, but could possibly recovered by other means ...`

Reply:
I'll take that.

```suggestion
   An object that has lost read quorum is effectively "lost", but could possibly be recovered through other means such as :ref:`replication resynchronization <minio-bucket-replication-resynchronize>`.
```